quote-order-bff.yml

```diff
@@ -1189,18 +1189,81 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/TripRateList'
-  /api/v1/quote-order-bff/triprates/vendor/{vendorId}/:
+  /api/v1/quote-order-bff/triprates/bidordercenter/{bidOrderCenterId}?pageNo=[pageNo]&pageSize=[pageSize]&site=[site]&shipper=[shipper]&consignee=[consignee]&pickupStartDate=[pickupStartDate]&pickupEndDate=[pickupEndDate]&deliveryStartDate=[deliveryStartDate]&deliveryEndDate=[deliveryEndDate]&saleRepresentative=[saleRepresentative]&status=[status]:
```

Comment:
Didn't we say that will call /api/v1/quote-order-bff/triprates and bidOrderCenterId or vendorId will be a parameters?
Cause we have the situation on the frontend that we don't know which bidOrderCenterId the user is part of.

Reply:
you are right, I had not removed it by accident. adding bidOrderCenterId as a query param.

Is there a scenario in which vendorId can be passed as a query param? (In tripRates vendorId is extracted though the identification client via the username)